Sprint-3/implement/is-proper-fraction.js

```diff
@@ -32,3 +32,21 @@
 // target output: false
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+function isProperFraction(numerator, denominator) {
+    // Check if the denominator is zero and throw an error if it is
+    if (denominator === 0) {
+        throw new Error("Denominator cannot be zero");
+    }
+    // Check if the numerator is less than the denominator and return true if it is
+    // Otherwise, return false
+    return numerator < denominator;
+}
+
+//Testing the function
+console.log(isProperFraction(2, 3)); //should return True
+console.log(isProperFraction(5, 2)); //should return False
+console.log(isProperFraction(-4, 7)); //should return True
```

Comment:
In mathematics, -4/7 == 4/-7, and -4/-7 == 4/7.
So, ideally `isProperFraction()` should recognise all of them as proper fractions.

Hint: If you compute the absolute value of both parameters inside the function first, the code can become much simpler.

Reply:
I added a part which first make absolute numbers then check them